src/libs/asyncOpenURL/index.website.ts

```diff
@@ -22,8 +19,13 @@ export default function asyncOpenURL(promise, url, shouldSkipCustomSafariLogic)
         const windowRef = window.open();
         promise
             .then((params) => {
-                windowRef.location = typeof url === 'string' ? url : url(params);
```

Comment:
Hey @luacmartins! I see you're the last person who modified this line. Is there a reason why you skipped `href` here? There was a type error without, so I added it and I couldn't spot any regression, but I would like you to cross check it.

Reply:
Hey @kowczarz , did you try this? It's working for me.

```ts
            .then((params) => {
                if (!windowRef) {
                    return;
                }
                windowRef.location = typeof url === 'string' ? url : url(params);
            })
```

Reply:
@kowczarz bump

Reply:
Oh, that must have been some typescript caches left, because now it works flawlessly.